core/common/buffer.cpp

```diff
@@ -0,0 +1,107 @@
+#include <utility>
+
+#include "buffer.hpp"
+#include "hexutil.hpp"
+
+namespace kagome::common {
```

Comment:
Does'nt our coding style assume that padding is needed inside the namespace?

Reply:
yes, it wasn't installed in clion by default for some reason